microsoft-edge/visual-studio-code/microsoft-edge-devtools-extension/get-started-right-click-html.md

```diff
@@ -0,0 +1,137 @@
+* Step 2: Start DevTools by right-clicking an HTML file
+* Step 3: Arrange tabs
+* Step 4: Edit CSS in DevTools, updating the .css file automatically
+* Step 5: Step through JavaScript code in the Debugger
+* Step 6: Close DevTools
+-->
+
+
+<!-- ====================================================================== -->
+## Step 1: Install DevTools and prerequisites
+
+1. If you haven't already, do the steps in [Installing the DevTools extension for Visual Studio Code](./install.md), and then continue below.  You don't need to install and start a web server for the present tutorial, but it's recommended.
+
+
+<!-- ====================================================================== -->
+## Step 2: Start DevTools by right-clicking an HTML file
+
+<!-- the equivalent section in Opening is more general than this section, which uses Demos repo -->
+
+Right-clicking an `.html` file in Visual Studio Code's **Explorer** is the main way to open DevTools when your webpage doesn't require running on a web server.
+
+*  Unlike the **Launch Instance** button, this approach opens DevTools in Debug mode.
+
+*  Unlike the **Launch Project** button that we'll use later, this approach doesn't require you to generate a `launch.json` file.
+
+
+We'll demonstrate by opening the **demo-to-do** web app:
+
+1. In Visual Studio Code, select **File** > **Open Folder**.
+
+1. Go to the directory where you cloned the Demos repo, open the specific directory for the **demo-to-do** app, such as `C:\Users\username\Documents\GitHub\Demos\demo-to-do\`, and then click the **Select Folder** button:
+
+   ![Open Folder: demo-to-do](./get-started-images/open-folder-demo-todo.png)
+
+1. Select **Activity Bar** > **Explorer** (![Explorer icon](./get-started-images/explorer-icon.png)) > right-click `index.html` and then select **Open with Edge** > **Open Browser with DevTools**:
+
+   ![Right-click > Open Browser with DevTools](./get-started-images/open-browser-with-devtools.png)
+
+   *  The **Edge DevTools** tab opens.
+
+   *  The **Edge DevTools: Browser** tab opens, displaying the webpage you right-clicked.
+
+   *  The Debug toolbar of Visual Studio Code opens, the **Debug Console** opens at the bottom, and the **Run** pane opens.  These features indicate that Visual Studio Code is in Debug mode:
+
+   ![The two Edge DevTools tabs and the Debug toolbar](./get-started-images/devtools-extension-v211.png)
+
+
+<!-- ====================================================================== -->
+## Step 3: Arrange tabs
+
+1. Toward the upper left corner of the **Edge DevTools** tab, click the **Toggle screencast** button a couple times:
+
+   ![Cross-toggling the two tabs on and off](./get-started-images/cross-toggling-tabs.png)
+
+1. In the top right of the **Edge DevTools: Browser** tab, click the **Close DevTools** or **Open DevTools** button a couple times.  The two DevTools tabs are automatically positioned as you open and close them from each other.
+
```

Comment:
This is a very confusing part, I am not sure that what you define here are actually features but happy accidents about VS Code arranging newly opened tabs. 
A much more useful explanation would be that the icon in the devtools allows you to toggle the browser preview panel and the icon in the browser preview panel allows you to toggle the developer tools. It would also make sense to tell people that they can dock the browser preview panel anywhere in VS code - meaning you can also use it under the source code one.

Reply:
Longer term, it probably makes more sense to move this into an own document. There we can explain how to customise the interface and how to use it - including the emulation options in the browser preview.

Reply:
This PR updates/expands the Device Emulation article.  Consider: How closely related is Device Emulation & arranging tabs?  There's now a Customize TOC bucket.  Is Device Emulation something for the Customize TOC bucket?

Reply:
Broke out steps & screenshots.